internal/featuregates/mapper.go

```diff
@@ -31,6 +34,7 @@ func NewMapper() *Mapper {
 	// Add your downstream to upstream mapping here
 	operatorControllerGates := map[configv1.FeatureGateName][]string{
 		// features.FeatureGateNewOLMMyDownstreamFeature: {MyUpstreamControllerOperatorFeature}
+		features.FeatureGateNewOLMCatalogdMetas: {APIV1MetasHandler},
```

Comment:
`APIV1MetasHandler` is a catalogd gate, not a operator-controller gate. Need to move this to the catalogdGates map.